data/reviewed-preprints/85111.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "85111",
+  "doi": "10.7554/eLife.85111",
+  "status": "reviewed",
+  "authorLine": "Rebecca Jordan, Georg B. Keller",
+  "title": "The locus coeruleus broadcasts prediction errors across the cortex to promote sensorimotor plasticity",
+  "indexContent": "The locus coeruleus broadcasts prediction errors across the cortex to promote sensorimotor plasticity",
+  "titlePrefix": "Title prefix",
+  "stage": "published",
+  "published": "2023-04-25T00:00:00Z",
+  "reviewedDate": "2023-04-25T00:00:00Z",
+  "versionDate": "2022-08-03T00:00:00Z",
+  "statusDate": "2022-08-01T00:00:00Z",
```

Comment:
Let's try these dates:

```
  "published": "2022-07-03T00:00:00Z",
  "reviewedDate": "2022-07-03T00:00:00Z",
  "versionDate": "2023-04-25T00:00:00Z",
  "statusDate": "2023-04-25T00:00:00Z",
```

That way it's original `published` date is before `29561` but it has a more recent `statusDate` which is more recent.